lib/src/app/ml_training.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:viam_sdk/protos/app/ml_training.dart';
+
+/// gRPC client for the [MlTrainingClient]. Used for working with ML training jobs.
+///
+/// All calls must be authenticated.
+class MlTrainingClient {
```

Comment:
`Ml` should be `ML`: https://dart.dev/effective-dart/style#do-capitalize-acronyms-and-abbreviations-longer-than-two-letters-like-words

2 Letters should still be fully capitalized